### PR TITLE
[WOR-552] update rawls-model dependency

### PR DESCRIPTION
**Pull request by ghost**
Ticket: https://broadworkbench.atlassian.net/browse/WOR-552
Orch parses the status response from rawls so had to update the rawls model dependency and fix tests  
- updated `rawls-model` dependency to use the new version (https://github.com/broadinstitute/rawls/pull/2121) which adds BPM and WSM to the rawls status check 
-  used workbench-libs version of`Notifications` instead of `rawls-model `
- updated akka version from `2.5.32` to `2.6.19` (same version as rawls) to fix `java.lang.NoSuchMethodError` when running unit tests

firecloud-orchestration Swagger : 
Screen Shot 2022-11-23 at 3 57 20 PM

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -41,7 +41,7 @@ object Dependencies {
     "com.typesafe.scala-logging"    %% "scala-logging"       % "3.9.5",
 
     "org.parboiled" % "parboiled-core" % "1.4.1",
-    excludeGuava("org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-e0584dbdc")
+    excludeGuava("org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-34e8799b-SNAP")
```
Comment (cahrens) [inline project/Dependencies.scala]: So does the Rawls PR have to merge first, and then the version number gets updated here?

Comment (ghost) [reply project/Dependencies.scala]: As far as I know, I have to merge this PR first then merge the Rawls PR and update with an official version here but I could be wrong

**Review by ehigham — APPROVED** [review]

File: project/Dependencies.scala [hunk 2]
```diff
@@ -41,7 +41,7 @@ object Dependencies {
     "com.typesafe.scala-logging"    %% "scala-logging"       % "3.9.5",
 
     "org.parboiled" % "parboiled-core" % "1.4.1",
-    excludeGuava("org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-e0584dbdc")
+    excludeGuava("org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-902d77f8-SNAP")
```
Comment (cahrens) [inline project/Dependencies.scala:44]: I think I already asked this, but just to be clear will ask again.

So the plan is that you merge the Rawls PR first and then update the version string here so it isn't a SNAP version?

Comment (cahrens) [reply project/Dependencies.scala:44]: Ah, I saw your previous response.

**Review by cahrens — APPROVED** [review]

**Pull request merged**